components/productList.html

```diff
@@ -0,0 +1,412 @@
+                    <button class="btn btn-primary mr-1">Login</button>
+                </a>
+                <div class="search-div flex-row align-center mobile-hidden mr-1">
+                    <label for="serach-box">
+                        <input class="input-search btn-border-none bg-transparent" id="serach-box" type="text" placeholder="Search" aria-label="Search Box"/>
+                        <button class="nav-search-btn btn-border-none bg-transparent" type="submit">
+                            <i class="fas fa-search"></i>
+                        </button>
+                    </label>
+                </div>
+                
+                <ul class="nav-link-items flex-row align-center justify-around">
+                    <li class="nav-items user-icon mobile-hidden">
+                        <a href="#">
+                            <i class="fas fa-user-alt"></i>
+                        </a>
+                    </li>
+                    <li class="nav-items nav-cta-icon">
+                        <a href="#">
+                            <i class="fas fa-shopping-cart"></i>
+                            <span class="icon-badge number-sm-badge">9</span>
+                        </a>
+                    </li>
+                    <li class="nav-items nav-cta-icon">
+                        <a class="#">
+                            <i class="fas fa-heart"></i>
+                            <span class="icon-badge number-sm-badge">9</span>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+
+        <aside  class="aside-box grid">
+            <div class="filter-sections filter-head flex-row align-center justify-between">
+                <span class="xsm-heading font-weight-5">Filter</span>
+                <button class="btn-border-none bg-transparent filter-cta-btn font-weight-5">Clear All</button>
+            </div>
+            <div class="filter-sections m-hr-1 flex-column">
+                <div class="flex-row align-center justify-between">
+                    <span class="sub-filt-head font-weight-5">Sort</span>
+                    <button class="btn-border-none bg-transparent filter-cta-btn font-weight-5">Reset</button>
+                </div>
+                <ul class="m-vl-1 filter-items">
+                    <li class="align-center">
+                        <label for="price-l-to-h">
+                            <input class="filter-inputs" type="radio" id="price-l-to-h" name="price-l-to-h" aria-label="Price Low to High">
+                            Price - Low to High
+                        </label>
+                    </li>
+                    <li>
+                        <label for="price-h-to-l">
+                            <input class="filter-inputs" id="price-h-to-l" type="radio" name="price-h-to-l"  aria-label="Price High to Low">
+                            Price - High to Low
+                        </label>
```

Comment:
display only one among them.
![Screenshot 2022-02-17 at 12 22 19 PM](https://user-images.githubusercontent.com/79623546/154421857-95525541-23c2-4f59-b32d-ba61304b7eb4.png)

Reply:
I mean both are selected at the same time.

Reply:
The remaining looks good, you can merge.

Reply:
```suggestion
                        </label>
```
```suggestion
                        </label>
```

That is functionality  which I will add in reactjs

Reply:
ok